en/includes/references/extend/user-mgt/provisioning/add-scim2-custom-user-schema-support.md

```diff
@@ -179,8 +179,7 @@ Let's see when we have a **deviceNames** simple multi-valued attribute.
 4. Click Finish.
 5. Go to the **Edit Attribute** of the custom attribute you just created.
 6. Select necessary options in **Attribute Configurations** to make the field visible in user profiles and click **Update**.
-7. Navigate to **Additional Properties** tab and enter following property and click **Update**.
-    - `multiValued`: `true`
+7. Check the **Allow multiple values for this attribute**.
```

Comment:
We maintain a single document for Is 7.1.0 and next. Hence any changes from here will reflect on 7.1.0 docs as well.
Since 7.1.0 doesn't have this capability, let's add a if condition to show the necessary option regarding the product version.

Reply:
Removed from the includes folder and duplicated three files into Asgardeo, IS 7.1 and next folders.
For Asgardeo the attribute properties related parts were removed.
For IS 7.1 the previous doc file was mainatained.
For next folder, the relevant multiValued related changes were applied (The same change applied for the Asgardeo folder also)